frontend/www/js/omegaup/arena/submissions.ts

```diff
@@ -45,56 +51,69 @@ export function submitRun(request: any): void {
     });
 }
 
-export function updateRun(
-  run: types.Run,
-  target: arena_ContestPractice | problem_Details,
-): void {
-  trackRun(run, target);
+export function updateRun(request: {
```

Comment:
```suggestion
export function updateRun({
    run,
    target,
  } : {
```

así no hay necesidad de hacer `request.run` y se puede usar `run` directo. igual en todos los demás lugares donde se usa `request: { ... }`.

Reply:
Listo.